### ISSUE TITLE
total size of new array must be unchanged

### ISSUE THREAD

**Issue by khoaprogrammer** [experiencor/basic-yolo-keras#13]
tried to build the model and error come up at this line:
output = Reshape((GRID_H, GRID_W, BOX, 4 + 1 + CLASS))(x)

the only thing i change is the LABELS, only one class. Please help!

And can you change the model to the old-style (create the Sequence and add layers one by one)? So we can watch it's summary whenever we want?

**Comment by experiencor** [comment]
It's fixed now. I forgot to change the number of filters to fit the number of classes.

The old style with the Sequence API is not a good idea in this case as the full YOLO model requires a skip connection, which is better implemented with the Functional API. You will find that the Functional API is much more expressive than the Sequence API later on when you deal with complicated architectures.

**Comment by khoaprogrammer** [comment]
thank you so much! do you keep the previous implementation of tiny yolo version?? i'm learning based on that so can you send me that? thank you so much again :)

**Comment by experiencor** [comment]
[yolo_keras.zip](https://github.com/experiencor/basic-yolo-keras/files/1346501/yolo_keras.zip)

This is the old code. Not tested so you should have to fix here and there.
